### DISCUSSION TITLE
An action stuck in progress in UI. I see it finished succesfully in logs.

### DISCUSSION THREAD

**Discussion by denist-huma** [community/community#15133] [Actions]
## expected

I expected to see a success report. My action did it job, from the log:

```sh
2022-04-18T10:47:46.0149671Z UPDATED RELEASES:
2022-04-18T10:47:46.0150266Z NAME       CHART                VERSION
2022-04-18T10:47:46.0150673Z ppserver   ../charts/ppserver     1.0.0
2022-04-18T10:47:46.0151156Z 
2022-04-18T10:47:46.0191043Z Post job cleanup.
2022-04-18T10:47:46.0563632Z running command: sudo kill 3926 || true
```

## actual

1. An action stuck in progress in UI https://github.com/huma-engineering/phoenix-server-deployments/runs/6061631632?check_suite_focus=true
2. I see it finished succesfully in logs. 
3. Deploy [dev] the time display of 34 mins is wrong. By log it took 2-3 minutes, from the log

```sh
2022-04-18T10:46:34.5610871Z Requested labels: ubuntu-latest
<>
2022-04-18T10:47:46.2937994Z Cleaning up orphan processes
```

**Comment by lebenitza** [comment]
This happened to me once as well, all the individual steps are finished and marked as so in the UI, the job itself is still stuck running for 45 minutes now (the steps totalled to ~3 min). Canceling the workflow does not do anything. Hopefully the CI minutes won't count towards the quota 🤞🏼

**Reply by j-mcgregor** [reply]
Did you find a fix for this?

**Reply by denist-huma** [reply]
this may be related with a frontend disconnected from backend event stream or something. Internet is not reliable WYSIWYG not :hankey: :

**Reply by lebenitza** [reply]
> Did you find a fix for this?

Not really, I just had to wait and the state somehow reconciled in the backend after a while. No weird amounts were added against the CI quota. Did not happen since then

**Reply by RichardTMiles** [reply]
My team is seeing this for the first time on a single run today.

**Reply by jontze** [reply]
> My team is seeing this for the first time on a single run today.

I experienced this as well today, in my case it was related to an incident and degraded performance with github actions, might be related to this: https://www.githubstatus.com/

**Comment by ghost** [comment]
This has been quite annoying. It happens both with Self-hosted runners and public runners. I've experienced over 5 m tasks on hold in the UI.

An easy way to replicate:
- Use docker buildx with multiple platforms.
- kubectl wait or rollout status to wait for a kubernetes deployment.

**Comment by shravanngoswamii** [comment]
I don't know why and how it's happening, but it's just a simple page build workflow, and it's stuck in progress, showing completed in report build system, I tried to cancel it, but it's not even cancelling, saying that failed to cancel workflow. Because of this, I am not able to run other page builds too!

Here is the link: https://github.com/shravanngoswamii/portfolio/actions/runs/11296926110

**Comment by allusernamestakenexceptthis** [comment]
I don't get why can't we force cancel it.  canceling seems not working.

**Comment by city41** [comment]
I'm seeing this today. All workflow steps finish, yet the overall workflow is still set to in progress. 

Today it's not too bad, the workflows seem to go to complete after about 5-10 minutes.

Also new workflows starting is taking quite some time, but I think for public actions you're limited to number of concurrent workflows and so the ones stuck in progress prevent new ones from starting.
